src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/TovStar.cpp

```diff
@@ -23,19 +23,15 @@ TovStar::TovStar(const double central_rest_mass_density,
     : central_rest_mass_density_(central_rest_mass_density),
```

Comment:
How about "Support multiple TOV solutions in one executable"? I don't think this is a "serious" bug. It hasn't resulted in us publishing incorrect results. I would argue that most bug fix PRs we get are much more serious than this. It's even threadsafe as far as I can tell, so you're really adding support for a new feature rather than fixing a bug...